docs/yamls/dapp.md

```diff
@@ -11,25 +11,31 @@
 | &rarr;Annotations    | Definitions that can describe characteristics of the dApp that later on can be used to process/group the dApps in your cluster.                                                             |
 | &rarr;parent         | Defines dApp context in relation to the cluster. For example `app1.app2` would mean that this dApp is located on the path `root->app1->app2->app-name`. It is injected by the Inspr daemon. |
-| &rarr;sha256         | Tags images with their sha256 digest.                                                                                                                                                       |
+| &rarr;uuid           | Universally unique identifier for the DApp.                                                                                                                                                 |
 | spec                 | specification of dApp                                                                                                                                                                       |
 | &rarr;Node           | Section describing the dApp Node                                                                                                                                                            |
 | &rarr;&rarr;Meta     | Metadata of the Node                                                                                                                                                                        |
 | name                 | Defines Node name                                                                                                                                                                           |
 | reference            | String that is utilized to define certain tags to the Node in question, a way for the user to categorize the numerous Nodes in the cluster.                                                 |
 | Annotations          | Definitions that can describe characteristics of the Node that later on can be used to process/group the Nodes in your cluster.                                                             |
 | parent               | Defines the Node context in relation to the cluster for example `app1.app2` would mean that this Node is located on the path `root->app1->app2`. It is injected by the Inspr daemon.        |
-| sha256               | tags images with their sha256 digest.                                                                                                                                                       |
+| uuid                 | Universally unique identifier for the Node.                                                                                                                                                 |
```

Comment:
As this is the YAML definition for dApps/Nodes, I believe there's no need for UUID here as the user will never set it's value